build/config.gni

```diff
@@ -8,7 +8,7 @@ declare_args() {
   # "nightly" for nightly channel release.
   # "" for stable channel release.
   brave_channel = ""
-  is_release_channel = true
+  is_release_channel = false
```

Comment:
this was wrong from https://github.com/brave/brave-core/commit/2cbcb14513fb3d23a06fe1a9c1336d9312886b23